packages/datadog-instrumentations/test/pg.spec.js

```diff
@@ -0,0 +1,265 @@
+const agent = require('../../dd-trace/test/plugins/agent')
+const dc = require('dc-polyfill')
+
+const clients = {
+  pg: pg => pg.Client
+}
+
+if (process.env.PG_TEST_NATIVE === 'true') {
+  clients['pg.native'] = pg => pg.native.Client
+}
+
+describe('pg instrumentation', () => {
+  withVersions('pg', 'pg', version => {
+    const queryClientStartChannel = dc.channel('apm:pg:query:start')
+    const queryPoolStartChannel = dc.channel('datadog:pg:pool:query:start')
+
+    let pg
+    let Query
+
+    function abortQuery ({ abortController }) {
+      const error = new Error('Test')
+      abortController.abort(error)
+
+      if (!abortController.signal.reason) {
+        abortController.signal.reason = error
+      }
+    }
+
+    before(() => {
+      return agent.load(['pg'])
+    })
+
+    describe('pg.Client', () => {
+      Object.keys(clients).forEach(implementation => {
+        describe(implementation, () => {
+          let client
+
+          beforeEach(done => {
+            pg = require(`../../../versions/pg@${version}`).get()
+            const Client = clients[implementation](pg)
+            Query = Client.Query
+
+            client = new Client({
+              host: '127.0.0.1',
+              user: 'postgres',
+              password: 'postgres',
+              database: 'postgres',
+              application_name: 'test'
+            })
+
+            client.connect(err => done(err))
+          })
+
+          afterEach(() => {
+            client.end()
+          })
+
+          describe('abortController', () => {
+            afterEach(() => {
+              if (queryClientStartChannel.hasSubscribers) {
+                queryClientStartChannel.unsubscribe(abortQuery)
+              }
+            })
+
+            describe('using callback', () => {
+              it('Should not fail if it is not aborted', (done) => {
+                client.query('SELECT 1', (err) => {
+                  done(err)
+                })
+              })
+
+              it('Should abort query', (done) => {
+                queryClientStartChannel.subscribe(abortQuery)
+
+                client.query('SELECT 1', (err) => {
+                  if (err && err.message === 'Test') {
+                    done()
+                    return
+                  }
+
+                  done(new Error('Query was not aborted'))
```

Comment:
Could it be something like that?
```suggestion
                  assert.propertyVal(err, 'message', 'Test')
                  done()
```